plugins/plugin-bash-like/src/pty/copy-out.ts

```diff
@@ -92,25 +90,3 @@ const copyRecursive = async (src: string, dest: string) => {
     return copy(src, dest)
   }
 }
-
-/** plain file copy that is ASAR-friendly */
-const copyFile = (src: string, target: string): Promise<string> => new Promise<string>(async (resolve, reject) => {
-  debug('copyFile', src, target)
-
-  let targetFile = target
-
-  // if target is a directory a new file with the same name will be created
-  if (await exists(target)) {
-    if ((await lstat(target)).isDirectory()) {
-      targetFile = join(target, basename(src))
-    }
-  }
-
-  fs.readFile(src, (err, data) => {
-    if (err) {
-      reject(err)
-    } else {
-      resolve(writeFile(targetFile, data))
-    }
-  })
-})
```

Comment:
@starpit, is copyFile just not in use, but may be useful?

Reply:
If you need it in the future you can use git, to avoid these unused pieces of code is one of the main reason of using git.

Reply:
agreed, we can remove it